File: .NET Core/Exceptions/NotFoundBackendException.cs
using com.contidio.sdk.proto;
using System.Net;

namespace Contidio.Sdk.Exceptions
{
    public class NotFoundBackendException : BackendException
    {
        public NotFoundBackendException(BackendErrorCode errorCode, string message)
            : base(errorCode, message, "NotFoundBackendException", HttpStatusCode.NotFound)
        {
        }
    }
}

File: Java/src/main/java/com/contidio/sdk/exceptions/NotFoundBackendException.java
package com.contidio.sdk.exceptions;

import org.apache.commons.httpclient.HttpStatus;

import com.contidio.sdk.proto.MetadataProtocol.BackendErrorCode;

@SuppressWarnings("serial")
public class NotFoundBackendException extends BackendException {
	public NotFoundBackendException(BackendErrorCode errorCode, String message) {
		super(errorCode, message, "NotFoundBackendException", HttpStatus.SC_NOT_FOUND);
	}
}
